dev/flake.nix

```diff
@@ -14,7 +14,8 @@
       ];
       perSystem = { pkgs, lib, config, ... }: {
         treefmt = {
-          projectRootFile = "flake.nix";
+          projectRoot = ../.;
```

Comment:
Is this legal in pure evaluation? AFAIU, a flake (or sub-flake) cannot refer to contents outside its containing directory.

Try running nix with sandbox enabled; I think it is disabled by default on macOS.

Reply:
i ran the command with sandbox on ubuntu. `treefmt` is able to scan 26 files.

<img width="1512" alt="Screenshot 2024-02-19 at 10 02 27 PM" src="https://github.com/juspay/services-flake/assets/46927550/e53681ec-6dbf-4276-9035-c42b1b4a74b2">